python/core/qgsmargins.sip

```diff
@@ -138,90 +138,14 @@ class QgsMargins
 
 
 bool operator==( const QgsMargins &lhs, const QgsMargins &rhs );
-%Docstring
- Returns \c true if ``lhs`` and ``rhs`` are equal; otherwise returns \c false.
- :rtype: bool
-%End
```

Comment:
Apparently not...

It would be great to have a look at 

 * core/qgsexpressioncontext.sip
 * core/qgsmargins.sip
 * core/qgsfeaturerequest.sip
 * core/qgsoptionalexpression.sip